src/cli.rs

```diff
@@ -108,6 +108,19 @@ pub enum Command {
         #[arg(long, value_parser, value_delimiter = ',')]
         systems: Vec<System>,
     },
+
+    /// Generates shell completion scripts
+    Completion {
+        #[arg(value_enum)]
+        shell: clap_complete::Shell,
+
+        /// Flake URL or github URL
+        ///
+        /// A specific nixci configuration can be specified
+        /// using '#': e.g. `nixci .#extra-tests`
+        #[arg(default_value = ".")]
+        flake_ref: FlakeRef,
```

Comment:
What is the significant of this argument? i.e., when the user runs `nixci completion fish github:srid/haskell-template` what does it mean?